src/MusicTheory/Accidental.elm

```diff
@@ -0,0 +1,14 @@
+module MusicTheory.Accidental exposing (Accidental(..))
+
+{-| An accidental describes by how many semitones a letter is raised or lowered.
+-}
+
+
+type Accidental
```

Comment:
We only need the accidental type for spelling. It's a must have IMO.

Reply:
See later comments as well.

Note that `fromTuple` is useful here: https://github.com/elm-music/music-theory/blob/feature/start/pitch-class-enharmonics/tests/PitchClassTests.elm#L119

This test should be moved to `SpellingTests`. Anyway it shows how it can be useful to be able to easily go back to a `PitchClass` representation from a `PitchClassSpelling` representation.